p/themes/base-theme/template.css

```diff
@@ -425,6 +425,16 @@ a.btn {
 	user-select: none;
 }
 
+#sidebar {
+	scrollbar-width: thin;
+	scrollbar-color: rgba(0, 0, 0, 0.0) rgba(0, 0, 0, 0.0);
```

Comment:
The Alpha value should be increased to be visible.

Reply:
Ok. But for some reason, the current code does not work in my tests with Firefox 65: the scrollbar never gets visible, even when hovering.

Reply:
I tested with https://www.mozilla.org/firefox/developer/

Reply:
I've been able to see the issue myself. Additionally, despite the fact that it was working last night in v64, it's not working today. I even rolled back to my first implementation in ab29de9 to ensure that it wasn't a regression caused by subsequent changes, and that's not the case.

Anyways, the issue seems to be that Firefox isn't picking up on the #sidebar:hover on page load. If you make any changes in the block in inspector view, it picks it up and works fine until the next page load.

Are you content with just making the native scrollbar not hide when the tree's being not being hovered over for now?

Reply:
Please see this quick attempt https://github.com/FreshRSS/FreshRSS/pull/2117/commits/ab9817e79377c0d28bac614870a794f119c5b068
But removing the `height: 90vh;` might have broken something else. I did not have time to check yet. Will have time again on Sunday.